Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php

```diff
@@ -111,6 +111,8 @@ public function getPlaceholderValues(int $userId, int $objId): array
         if (!$user instanceof ilObjUser) {
             throw new ilException('The entered id: ' . $userId . ' is not an user object');
         }
+        $this->language = new ilLanguage($user->getLanguage());
+        $this->language->loadLanguageModule('certificate');
```

Comment:
I suggest to use a local variable `$usr_language` for this, instead of overwriting the member variable. We should rename the member variable to `actor_language`.